crud/ProductManager/ProductManager/Classes/Order.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace ConsoleApplication1.Classes
+{
+    public class Order
+    {
+        public int Id { get; set; }
+        public DateTime CreateDate { get; set; }
+        public DateTime UpdatedDate { get; set; }
+        public int ProductId { get; set; }
+        public int Status { get; set; }
```

Comment:
Status should be Enum type. Let's create new OrderStatusEnum class and use it as a Status